dotcom-rendering/tsconfig.json

```diff
@@ -65,10 +65,13 @@
         // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
         "forceConsistentCasingInFileNames": true,
         "resolveJsonModule": true,
-        "preserveConstEnums": true
+        "preserveConstEnums": true,
         /* Experimental Options */
         // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
         // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-    },
-    "exclude": ["cypress"]
+		"skipLibCheck": true
```

Comment:
The `esbuild-loader` package contains `ts` errors which don't pass the commit hook, using `skipLibCheck` allows us to push the branch though would need a better long term solution.